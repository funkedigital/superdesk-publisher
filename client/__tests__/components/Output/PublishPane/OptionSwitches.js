import React from "react";
import OptionSwitches from "../../../../components/Output/PublishPane/OptionSwitches";
import { render } from "@testing-library/react";

const destination = {
  is_published_fbia: true,
  paywall_secured: false
};

describe("Output/PublishPane/OptionSwitches", () => {
  it("renders properly", async () => {
    const { container } = render(
      <OptionSwitches
        fbiaEnabled={true}
        paywallEnabled={true}
        destination={destination}
        onChange={jest.fn()}
      />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
