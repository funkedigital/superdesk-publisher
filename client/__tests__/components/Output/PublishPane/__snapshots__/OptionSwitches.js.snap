// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output/PublishPane/OptionSwitches renders properly 1`] = `
<div
  class="form__row"
>
  <span
    sd-tooltip="Publish to facebook"
  >
    <span
      class="sd-check__wrapper"
    >
      <span
        class="sd-checkbox checked"
      />
      <label
        class=""
        style="color: rgb(255, 255, 255);"
      >
        Facebook
      </label>
    </span>
  </span>
  <span
    sd-tooltip="Enable Paywall"
    style="margin-left: 2em;"
  >
    <span
      class="sd-check__wrapper"
    >
      <span
        class="sd-checkbox"
      />
      <label
        class=""
        style="color: rgb(255, 255, 255);"
      >
        Paywall Secured
      </label>
    </span>
  </span>
</div>
`;
